IM_calculation/Advanced_IM/Models/Spear_3D_Model/run.py

```diff
@@ -146,13 +197,13 @@ def calculate_geom(output_dir, im_name):
     output_dir: folder that contains adv_im_comp.csv. used to grab data from 000, 090.
     im_name: adv_im model name
     """
-    df_000 = pd.read_csv(
-        os.path.join(output_dir, f"{im_name}_000.csv"), dtype={"component": str}
-    )
-    df_090 = pd.read_csv(
-        os.path.join(output_dir, f"{im_name}_090.csv"), dtype={"component": str}
-    )
-    #
+
+    c000_path = os.path.join(output_dir, f"{im_name}_000.csv")
```

Comment:
It appears that you chose to go with NS, EW because they are more self-explanatory than 000 and 090, but there are places in code still sticking to 000 and 090. Have you considered keeping things consistent?

Reply:
Good point, I've changed it back